RUNC.md

```diff
@@ -13,7 +13,7 @@ From within your `opencontainers/runc` repository run:
 ### apparmor
 
 ```bash
-make BUILDTAGS='seccomp apparmor' && sudo make install
+make BUILDTAGS='seccomp apparmor selinux' && sudo make install
```

Comment:
given the flow of this document, I'm not sure it makes sense to add it here. The next section guides a general user on how to build `runc` for SELinux; this section is providing the buildttags for AppArmor.

Reply:
Fixed. thanks